frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Page-Smoke-Metrics-ODC-4116.feature

```diff
@@ -0,0 +1,177 @@
+Feature: Metrics Tab on Monitoring Page
+    As a user, I should be able to run queries to check the usage
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke
+Scenario: CPU Usage Query on Metrics tab: MH-01-TC04
+    Given user is at Developer perspective
+    When user opens Monitoring page
```

Comment:
You could use 'user is on Monitoring Page` in the Given statement as well

Reply:
I removed the current given statement and replaced it with
"Given user is on Monitoring page"